pyseir/deployment/demographic_mapper.py

```diff
@@ -0,0 +1,592 @@
+        hospital_rate_icu_vent: np.array
+            Age specific per-capita rate of admission to ICU with ventilator.
+        """
+
+        hospital_rate_general = \
+            (self.parameters['hospitalization_rate_general']
+           - self.parameters['hospitalization_rate_icu']) / self.parameters['symptoms_to_hospital_days']
+        hospital_rate_icu = self.parameters['hospitalization_rate_icu'] / self.parameters['symptoms_to_hospital_days']
+        hospital_rate_ventilator = hospital_rate_icu * self.parameters['fraction_icu_requiring_ventilator']
+
+        return hospital_rate_general, hospital_rate_icu, hospital_rate_ventilator
+
+    def _reconstruct_recovery_rates_trajectory(self):
+        """
+        Reconstruct trajectory of inferred per-capita rates of recovery
+        during hospitalization through time from MLE model parameters and
+        compartments.
+
+        Returns
+        -------
+        hospital_recovery_rate_general: np.array
+            Age-specific rate of recovery in non-ICU.
+        hospital_recovery_rate_icu: np.array
+            Age-specific rate of recovery in ICU.
+        hospital_recovery_rate_icu_vent: np.array
+            Age-specific rate of recovery in ICU with ventilator.
+        """
+        mortality_rate_general, mortality_rate_icu, mortality_rate_icu_vent = \
+            self._reconstruct_mortality_rates_trajectory()
+
+        hospital_recovery_rate_general = (1 - mortality_rate_general) / \
+            self.parameters['hospitalization_length_of_stay_general']
+        hospital_recovery_rate_icu = (1 - mortality_rate_icu) * (1 - self.parameters['fraction_icu_requiring_ventilator']) \
+            / self.parameters['hospitalization_length_of_stay_icu']
+        hospital_recovery_rate_icu_vent = \
+            (1 - np.maximum(mortality_rate_icu, self.parameters['mortality_rate_from_ICUVent'])) \
+            / self.parameters['hospitalization_length_of_stay_icu_and_ventilator']
+
+        return hospital_recovery_rate_general, hospital_recovery_rate_icu, hospital_recovery_rate_icu_vent
+
+    def _age_specific_hospitalization_rates_among_infected(self, measure_unit):
+        """
+        Calculate age specific hospitalization rates for a given measure unit.
+
+        When unit is per capita, calculates the probability that an infected
+        person (asymptomatic or symptomatic) ultimately get admitted to
+        hospital:
+            probability of being symptomatic * rate of hospitalized / (rate of
+            hospitalized + rate of recovery without hospitalization)
+
+        When unit is per capita day, calculate the probability that an
+        infected person get admitted to hospital per day:
+           new hospitalization / (asymptomatic + symptomatic infections)
+
+        Parameters
+        ----------
+        measure_unit: CovidMeasureUnit
+            Unit of covid measure
+
+        Returns
+        -------
+          : np.array
```

Comment:
Would be nice to label these outputs since there are multiple fo them,

Reply:
Agreed, now all output variables are put in a single dictionary.